src/zeek-setup.cc

```diff
@@ -660,6 +660,16 @@ SetupResult setup(int argc, char** argv, Options* zopts)
 	init_net_var();
 	run_bif_initializers();
 
+	// Assign the script_args for command line processing in Zeek scripts.
+	if ( ! options.script_args.empty() )
+		{
+		auto script_args_val = zeek::id::find_val("zeek_script_args")->AsVectorVal();
```

Comment:
```suggestion
		auto script_args_val = id::find_val<VectorVal>("zeek_script_args");
```

Doesn't really matter in this case, just pointing out as "you can do it this way" for convenience and/or habit of avoiding raw pointers.

Reply:
I'll pull that change in.  I didn't realize that I could do the type handling that way now.  Thanks!